frontend/components/TooltipWrapper/_styles.scss

```diff
@@ -1,92 +1,25 @@
 .component__tooltip-wrapper {
-  display: inline-flex;
-  position: relative;
-
-  &:hover {
-    .component__tooltip-wrapper__tip-text {
-      visibility: visible;
-      opacity: 1;
-    }
-
-    .delayed-tip {
-      transition: 300ms all;
-      transition-delay: 300ms;
-    }
+  &__underline {
```

Comment:
@jacobshandling I just realized you're working on this. I found this weird behavior if the text goes in multiple lines #14526. Can we use `text-decoration-line` instead of the `border-bottom`? Here's the [example](https://codesandbox.io/s/interesting-panini-f34gpj?file=/index.html)

Reply:
Hi @marko-lisica – it seems like`text-decoration-line` can only give us a solid underline, while this component needs a dashed one.

This problem might be unique to this location (the QuerySidepanel), and looks like it is being caused by [this line](https://github.com/fleetdm/fleet/blob/53b08bfa9915232213dddff5ee5a382ba7d287ce/frontend/components/side_panels/QuerySidePanel/QueryTableColumns/ColumnListItem/_styles.scss#L22). It can be solved by telling the line not to wrap, but it seems @ghernandez345 needed to wrap it here.

If this does end up being something we want to address in the global component (this PR, which is a huge refactor and not exactly meant to address any individual bugs) and not in that specific place, we can look at it down the line. I'm sure it's possible to get that working how we want it one way or another.
 
Regardless, the best approach is to let this go through the normal bug process where the appropriate folks can have a look at it, prioritize it, assign an engineer at the right time.